tlsx/termination.go

```diff
@@ -0,0 +1,91 @@
+// Copyright © 2025 Ory Corp
+// SPDX-License-Identifier: Apache-2.0
+
+package tlsx
+
+import (
+	"net"
+	"net/http"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/urfave/negroni"
+
+	"github.com/ory/herodot"
+	"github.com/ory/x/healthx"
+	"github.com/ory/x/logrusx"
+	"github.com/ory/x/prometheusx"
+)
+
+type dependencies interface {
+	logrusx.Provider
+	Writer() herodot.Writer
+}
+
+func EnforceTLSRequests(d dependencies, allowTerminationFrom []string) (negroni.Handler, error) {
+	networks := make([]*net.IPNet, 0, len(allowTerminationFrom))
+	for _, rn := range allowTerminationFrom {
+		_, cidr, err := net.ParseCIDR(rn)
```

Comment:
Nit:

```suggestion
		_, network, err := net.ParseCIDR(rn)
```
`cidr` is the input, not the output.